index.html

```diff
@@ -288,15 +288,16 @@ <h3 class="titleA">About My self</h3>
             <h2>Contact me</h2>
             <p>If you have an application you are 
               interested in developing, a feature that you 
-              need built or a project that needs coding. I’d love 
+              need built or a project that needs coding.<br> I’d love 
               to help with it!</p>
+      </div>
 
 
-          <form action="https://formspree.io/f/mjvqvayk" method="post">
+          <form action="https://formspree.io/f/xpzgwrza" method="post">
 
-            <input type="User name" maxlength="30" required="required" placeholder="Enter Full Name">
+            <input type="User name" maxlength="30" required="required" placeholder="Enter Full Name"><label for="">User name:</label>
```

Comment:
Nice job adding the maxlength properties as requested by the previous reviewer👍🏼. but just as the previous reviewer has requested, the type attribute should be text not "User name" as there is no such attribute😉.

`type="text"`😉 

also, just as the previous reviewer stated, you can just use  only `required` and no need for `required="required"`😉. it doesn't take any value.